### DISCUSSION TITLE
Github Student Developer Validity

### DISCUSSION THREAD

**Discussion by Nikhil-Nishad** [community/community#41834] [GitHub Education]
I am new to Github (this is my temporary account), I have my student email id and I want to avail student developer pack. I am unsure about few things

1. Should I create my new account with .edu mail?
2. What will happen to my account, if I lost access to my email (after completing my graduation)?
3. Is there a way that with which I can change my registered email address later on?

**Comment by SilasRodrigues19** [comment]
@Nikhil-Nishad Hi.

1. You can use multiple email address, including the .edu
2. Probably lose the Developer Pack as he will no longer be a student
3. Yes. Access [this link](https://github.com/settings/emails)
